src/Command/PluginManager.php

```diff
@@ -36,6 +36,7 @@ protected function generate(array &$vars, Assets $assets): void {
 
     $discovery_types = [
       'annotation' => 'Annotation',
+      'attribute' => 'Attribute',
```

Comment:
Should hide this option for Drupal 10.1?

Reply:
I think no. D10.1 will have EOL this year, and besides seems this discovery is not breaking existing 10.1 sites.